torchmultimodal/utils/common.py

```diff
@@ -21,6 +21,37 @@ def get_current_device():
         return torch.device("cpu")
 
 
+def get_extended_attention_mask(attention_mask: Tensor) -> Tensor:
```

Comment:
I am not sure I would put this in utils/common, it's too specific to attention. I think for this PR it's fine to leave it here but we should consider a better location for this, such as layers/attention or utils/attention